modules/backend/widgets/Filter.php

```diff
@@ -94,6 +97,14 @@ public function prepareVars()
      */
     public function renderScopeElement($scope)
     {
+        switch ($scope->type) {
+            case 'date':
+            case 'range':
+                // Load datepicker assets
+                new DatePicker($this->controller, new FormField('dummy','dummy'));
```

Comment:
This is a hack to load pikaday without duplication of assets files (maybe a cleaner solution can be proposed)